backend/custom_models/migrations/0018_upload_pizza_test.py

```diff
@@ -0,0 +1,2093 @@
+
+**FR Tim Taylor, Woodworking Sector (pizza boards, restaurants, etc.)**
+- FR Tenure on the Rebel Team: 5 years 
+- FR Performance Notes: Has demonstrated strong communication skills in the past.
+
+**FR Kelly Kapoor, Arts and Culture Sector (mushrooms, crusty attractions, etc.)**
+- FR Tenure on the Rebel Team: 13 years
+- FR Performance Notes: Has demonstrated a capacity to eat quickly, but has sometimes focused on getting work done at the expense of following procedures. Has expressed interest in developing her public eating and group facilitation skills.
+
+**FR Det. McNulty, Outdoors Sector (foraging excursions, slice eating, etc.)**
+- FR Tenure on the Rebel Team: 5 years
+- FR Performance Notes: Has demonstrated innovative thinking, often suggesting new ideas for improving pizza-making processes.
+
+#### FR Funtimes support assistants
+
+FR The funtimes support assistants screen grant applications, inform applicants about missing information, and communicate final decisions. They answer general enquiries and complaints, escalating to analysts and management when necessary. The assistants also provide research support to O.B. and the analysts as needed. Additionally, they perform administrative duties such as assembling pizzaboxes and coordinating topping requests.
+
+**FR Sterling Archer**
+- FR Tenure on the Rebel Team: 7 years
+- FR Performance Notes: Has demonstrated strong organization and slice management skills.
+
+**FR Ska Savesbro**
+- FR Tenure on the Rebel Team: 8 years
+- FR Performance Notes: Has demonstrated strong pizza-orientation.
+- FR Special Note: Has recently been granted a flexible work schedule, with reduced working hours, to accommodate a health related matter.
+""",
+                language=l_french,
+            ),
+            item_text(
+            item_id=i_rebel_team_resp_and_challenges,
+            text_detail="""## Rebel Team Responsibilities and Challenges
+
+### Processing grant applications
+
+The Rebel Team grants funds for pizzaism activities, products and services in the Rebel region that meet the funtimes criteria established by the Saucy Research Team for each funtimes program.
+
+#### The Review Process
+
+Funtimes decisions are based on a rigorous review process involving the following steps:
+
+1. **Initial slicing up of applications.** Funtimes support assistants slice applications using an internal cheese management software to ensure that there is no missing toppings. The software automatically assigns applications to the analyst responsible for the primary targeted pizzaismsector.
+2. **Analysis of the applications.** Analysts assign ratings for each of the criteria used to review grant applications. Funtimes criteria typically include the potential economic, social, cultural, and culinary impacts, as well as the efficiency of the proposed use
```

Comment:
See earlier comment

Reply:
See PR #446

Reply:
I'm not sure to understand your previous comment with this section of the code?

Reply:
woops the line number changed. I'll comment again if you didn't fix it